rest-api/offline/biobank_samples_pipeline.py

```diff
@@ -340,7 +352,11 @@ def _get_hpo_type_sql(hpo_alias):
     ISODATE[MAX(created)] 'Sample Family Create Date',
     TIMESTAMPDIFF(HOUR, MAX(collected), MAX(created)) elapsed_hours,
     GROUP_CONCAT(DISTINCT biospecimen_kit_id) biospecimen_kit_id,
-    GROUP_CONCAT(DISTINCT fedex_tracking_number) fedex_tracking_number
+    GROUP_CONCAT(DISTINCT fedex_tracking_number) fedex_tracking_number,
+    GROUP_CONCAT(DISTINCT is_native_american) is_native_american,
```

Comment:
Not sure I understand the `DISTINCT` usage here; aren't there only two values `Y`, `N`?

Reply:
The current report returns a single row for a given (participant, biobank sample test) pair; there may be multiple biobank sample orders or biobank stored samples for that combination. So you might get multiple 'Y' or 'N' values in the inner query; DISTINCT makes sure we only return one value.

We are going to change this soon so we're matching individual order samples with individual stored samples (see [DA-418]), and we won't need this GROUP_CONCAT business anymore. But not in this PR.